pkg/agent/openflow/pipeline.go

```diff
@@ -881,11 +917,15 @@ func (c *client) establishedConnectionFlows(category cookie.Category) (flows []b
 		Action().GotoTable(egressDropTable.GetNext()).
 		Cookie(c.cookieAllocator.Request(category).Raw()).
 		Done()
-	cnpEgressEstFlow := c.pipeline[cnpEgressRuleTable].BuildFlow(priorityTopCNP).MatchProtocol(binding.ProtocolIP).
-		MatchCTStateNew(false).MatchCTStateEst(true).
-		Action().GotoTable(egressDropTable.GetNext()).
-		Cookie(c.cookieAllocator.Request(category).Raw()).
-		Done()
+	var cnpEgressFlows []binding.Flow
```

Comment:
As we already know the length of the slice, could we initialize the slice's capacity to be a little more efficient? so do below.

Reply:
will change this